### PR TITLE
modify test.go and source code in model Module

### PR DESCRIPTION
**Pull request by ludete**

### CODE REVIEW

**Comment by ludete** [comment]
modify test.go And source code in model module

File: model/BlockChain_test.go [hunk 1]
```diff
@@ -190,7 +191,14 @@ func TestBlockChain_BestBlockHash(t *testing.T) {
 		t.Error(err)
 		return
 	}
-	t.Logf("hash : %v, height : %d\n", hash, height)
+
+	if !bytes.Equal(hash[:], testBlcokChain.LastBlock.Hash[:]) {
+		t.Error("The two slice should be eqaul")
```
Comment (jjz) [inline model/BlockChain_test.go]: Error description information should be described in detail , including error methods and reasons for error

File: model/BlockChain_test.go [hunk 2]
```diff
@@ -190,7 +191,14 @@ func TestBlockChain_BestBlockHash(t *testing.T) {
...
+	}
+	if testBlcokChain.LastBlock.Height != height {
+		t.Error("the two height should be equal")
+	}
+
 }
 
 func TestBlockChain_SkipBlock(t *testing.T) {
```
Comment (jjz) [inline model/BlockChain_test.go]: Please use the unified method ,  Camel Case , not including _

**Comment by jjz** [comment]
ACK

**Pull request merged**